module/VuFind/src/VuFind/Captcha/Image.php

```diff
@@ -38,6 +38,26 @@
  */
 class Image extends LaminasBase
 {
+    /**
+     * Base URL of cache where image will be stored, e.g. /vufind/cache/
```

Comment:
It's a nitpick, but I'd call this a base path rather than a URL. :-)

Reply:
Done!